packages/block-editor/src/hooks/style.js

```diff
@@ -156,6 +158,7 @@ export const withBlockControls = createHigherOrderComponent(
 		return [
 			<TypographyPanel key="typography" { ...props } />,
 			<ColorEdit key="colors" { ...props } />,
+			<DimensionsPanel key="width" { ...props } />,
```

Comment:
Related to the suggestion around mirroring the existing organisation of the typography tools. Perhaps this key could then be:

```suggestion
			<DimensionsPanel key="dimensions" { ...props } />,
```